CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing Guidelines
+
+The Microsoft acs-engine project accepts contributions via GitHub pull requests. This document outlines the process to help get your contribution accepted.
+
+## Contributor License Agreements
+
+We'd love to accept your patches! Before we can take them, we have to jump a
+couple of legal hurdles.
+
+The [Microsoft CLA](https://cla.microsoft.com/) must be signed by all contributors. Please fill out either the individual or corporate Contributor License Agreement (CLA). Once you are CLA'ed, we'll be able to accept your pull requests. Once you are CLA'ed, we'll be able to accept your pull requests.
+
+***NOTE***: Only original source code from you and other people that have
+signed the CLA can be accepted into the repository.
+
+
+## Support Channels
+
+Whether you are a user or contributor, official support channels include:
+
+GitHub issues:
+- ACS-Engine: https://github.com/Azure/acs-engine/issues - file issues and PRs related to ACS-Engine
+- ACS: https://github.com/Azure/acs/issues - file issues and PRs related to Azure Container Service
```

Comment:
Is it worth establishing a weekly meeting cadence for the community to connect with?